app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] It would be nice to enable   `:confirmable` in your `user.rb`, so when users signup they are required to confirm their email befroe they can login suceesfully 👍🏿 , a prompt willl be shown to the user after they signup  to confirm their email before they can see the app 👍🏿

Reply:
@prolajumokeoni thank you for the feedback
it is resolved now